pkgs/overlay.nix

```
@@ -123,6 +123,12 @@ builtins.mapAttrs (_: patchPhps (fetchpatch {
 
     patches = [
       ./frr/0001-Don-t-throw-error-when-log-directory-already-exists.patch
+      # issue with best path selection erroneously deleting local mac
+      # addresses during incoming guest migration.
+      (fetchpatch {
+        url = "https://github.com/FRRouting/frr/commit/c3e264e27ac5b682c4647aeeaf4e61f09d5243f9.patch";
```

Note: `.patch` URIs are not always stable over longer times, better prefer `.diff` urls. One issue is that git(hub) auto-scales the lenth of the revision specifier with the growing number of commits in the repo. https://github.com/NixOS/nixpkgs/issues/257446#issuecomment-1736563091

For github, the following helps:
```suggestion
        url = "https://github.com/FRRouting/frr/commit/c3e264e27ac5b682c4647aeeaf4e61f09d5243f9.patch?full_index=1";
```

For gitlab, `.diff` URIs shall be preferred.

As this is only a temporary patch and will be pulled from the next upstream release instead, this is a purely informational note with no need to change the URI.